components/CommentModal/CommentModal.js

```diff
@@ -49,11 +82,24 @@ My thoughts:
     testimony && testimony.content ? testimony.content : defaultTestimony
 
   const EmailToMyLegislators = () => {
```

Comment:
Similar to [this discussion](https://github.com/codeforboston/advocacy-maps/pull/268#issuecomment-1062213141), we should avoid defining components inside other components.

We can fix this easily by renaming this and `EmailToCommittee` to `renderEmailToCommittee` and `renderEmailToMyLegislators`, and then calling the functions rather than rendering a component:

```
              <div>
                {renderEmailToMyLegislators()}
              </div>
              <div>
                {renderEmailToCommittee()}
              </div>

```

Reply:
I am thinking of moving the Email components to separate files, so they won't be defined inside other components.  Also, CommentModal is getting long, and this would make it a little easier to manage. 

I am thinking something like
```
const EmailToCommittee = ({
  checkedSendToCommittee,
  setCheckedSendToCommittee
}) => {
  const handleChangeSendToCommittee = () => {
    setCheckedSendToCommittee(!checkedSendToCommittee)
  }

  return (
    <div className="form-check">
      <input
        className="form-check-input"
        type="checkbox"
        checked={checkedSendToCommittee}
        id="flexCheckChecked"
        onChange={handleChangeSendToCommittee}
      />
      <label className="form-check-label" htmlFor="flexCheckChecked">
        Send copy to relevant committee
      </label>
    </div>
```